CHANGELOG.md

```diff
@@ -93,6 +93,10 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Made `clone` unsafe, like `fork`.
   ([#1993](https://github.com/nix-rust/nix/pull/1993))
 
+### Fixed
```

Comment:
The change log should be put under [\[Unreleased\]](https://github.com/nix-rust/nix/blob/master/CHANGELOG.md#unreleased---releasedate)

Reply:
done